stdlib/3/typing.pyi

```diff
@@ -260,6 +260,8 @@ class MutableSequence(Sequence[_T], Generic[_T]):
     def __delitem__(self, i: slice) -> None: ...
     # Mixin methods
     def append(self, object: _T) -> None: ...
+    def clear(self) -> None: ...
```

Comment:
I see `clear` in https://github.com/python/cpython/blob/master/Lib/_collections_abc.py, but not `copy`. Why should `copy` be here?

Reply:
@JelleZijlstra I've decided to put `copy` after reading this https://docs.python.org/3/library/stdtypes.html#mutable-sequence-types

Reply:
I'd be inclined to follow the implementation rather than the documentation here.

Reply:
@JelleZijlstra Agree